src/main/java/org/datadog/jmxfetch/Connection.java

```diff
@@ -18,6 +18,7 @@
 import javax.management.AttributeNotFoundException;
 import javax.management.InstanceNotFoundException;
 import javax.management.IntrospectionException;
+import javax.management.MBeanInfo;
 import javax.management.MBeanAttributeInfo;
 import javax.management.MBeanException;
```

Comment:
```suggestion
import javax.management.MBeanAttributeInfo;
import javax.management.MBeanInfo;
import javax.management.MBeanException;
```
Just to fix the CI linter test.